sql/expression/function/convert_tz.go

```diff
@@ -173,6 +184,22 @@ func getDeltaAsDuration(d string) (time.Duration, error) {
 	return time.ParseDuration(symbol + hours + "h" + mins + "m")
 }
 
+func getSystemDelta() string {
+	t := time.Now()
+	_, offset := t.Zone()
+	seconds := offset % (60 * 60 * 24)
```

Comment:
It looks like this is removing any full days from the seconds offset? Is this strictly necessary? I'm wondering if we should just honor the offset even if it's greater than `(60 * 60 * 24)`.